bot/exts/evergreen/trivia_quiz.py

```diff
@@ -261,44 +511,65 @@ async def declare_winner(channel: discord.TextChannel, player_data: dict) -> Non
 
                 winners_mention = " ".join(winner.mention for winner in winners)
             else:
-                word = "You"
                 author_index = list(player_data.values()).index(highest_points)
                 winner = list(player_data.keys())[author_index]
                 winners_mention = winner.mention
 
             await channel.send(
                 f"Congratulations {winners_mention} :tada: "
-                f"{word} have won this quiz game with a grand total of {highest_points} points!"
+                f"You have won this quiz game with a grand total of {highest_points} points!"
```

Comment:
Why has this change being done?

Reply:
you is you, no matter the plurality. it's english